index.js

```diff
@@ -5,5 +5,5 @@
 module.exports = function(content) {
 	this.cacheable && this.cacheable();
 	this.value = content;
```

Comment:
Another note. `this.cacheable` is set by default now so it would be possible to drop that.